### PR TITLE
Code snippet fix to work across desktops and mobiles

### PR DESCRIPTION
**Pull request by tayal1996**
https://browserstack.atlassian.net/browse/ACE-2558
Task Description: Nightwatch sample test cases to work across desktops and mobiles. Old scripts do not work on mobiles and ie browsers. With the changes, sample test script would pass across desktops and mobiles.

### CODE REVIEW

File: tests/single/single_test.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+    browser.url("https://www.google.com");
+    browser.setValue("input[name=q]", ["BrowserStack", browser.Keys.ENTER]); // this submits on desktop browsers
+    browser.pause(1000);
+    browser.title((result) => {
+      if (!/BrowserStack/i.test(result.value)) {
+        browser.submitForm("input[name=q]"); // this helps in mobile browsers
+      }
+    });
+    browser.expect.title().to.match(/BrowserStack/i);
+    browser.end();
```
Comment (rchougule) [inline tests/single/single_test.js:12]: any difference in chaining the commands like earlier as compared to writing them separately here? Any behavioural changes?

Or is it just convention/practice dependant?

Comment (tayal1996) [reply tests/single/single_test.js:12]: .expect.title().to.match(/BrowserStack/i); for this line to work it had to be individual that's why everyline was made individual for consistent

File: tests/single/single_test.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
       .pause(1000)
-      .assert.title('BrowserStack - Google Search')
-      .end();
-  }
+      .title((result) => {
+        if (!/BrowserStack/i.test(result.value)) {
+          browser.submitForm("input[name=q]"); // this helps in mobile browsers
+        }
+      })
+      .expect.title().to.match(/BrowserStack/i);
```
Comment (rchougule) [inline tests/single/single_test.js:12]:
```suggestion
      }).expect.title().to.match(/BrowserStack/i);
```
chained commands with an already ending block in the previous line can be written the following way.

Comment (rchougule) [reply tests/single/single_test.js:12]: similarly at other places if any

**Review by rchougule — APPROVED** [review]
LGTM

**Pull request merged**